ci-operator/jobs/open-cluster-management/registration-operator/open-cluster-management-registration-operator-master-periodics.yaml

```diff
@@ -0,0 +1,37 @@
+periodics:
+- agent: kubernetes
+  cluster: api.ci
+  cron: '@hourly'
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: open-cluster-management
+    repo: registration-operator
+  name: periodic-open-cluster-management-registration-operator-fast-forward
+  spec:
+    containers:
+    - args:
+      - --config-dir=./ci-operator/config/open-cluster-management/registration-operator/
```

Comment:
There is no such dir in `open-cluster-management/registration-operator`

Reply:
You will need to add the `openshift/release@master` in the extra refs. Also, I don't think that you need the existing extra_refs you have now.

Reply:
i want to fast-forward on https://github.com/open-cluster-management/registration-operator, not openshift actually. Will I be able to do that?

Reply:
The `./ci-operator/config/open-cluster-management/registration-operator/` config exists in `openshift/release` repo. So you need to clone that one instead.

Reply:
ah, i see. Updated. How could this job know which repo to fast-forward?  By reading zz_generated_metadata in config yaml?

Reply:
The tool will read all the configuration files that exist in the specified `--config-dir`